metrics/grafana/ticdc.json

```diff
@@ -14,7 +14,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "6.1.6"
+      "version": "7.5.7"
```

Comment:
Will this cause compatibility problem on older versions of Grafana?

Reply:
Something changes from `table` to `table-old` in the file, so I guess it is *not* compatible.